pkg/controller/flink/flink.go

```diff
@@ -206,21 +206,24 @@ func (f *Controller) CreateCluster(ctx context.Context, application *v1alpha1.Fl
 	newlyCreatedJm, err := f.jobManager.CreateIfNotExist(ctx, application)
 	if err != nil {
 		logger.Errorf(ctx, "Job manager cluster creation did not succeed %v", err)
-		f.LogEvent(ctx, application, corev1.EventTypeWarning,
-			fmt.Sprintf("Failed to create job managers: %v", err))
+		f.LogEvent(ctx, application, corev1.EventTypeWarning, "CreateClusterFailed",
+			fmt.Sprintf("Failed to create job managers for deploy %s: %v",
+				HashForApplication(application), err))
```

Comment:
HashForApplication() returns a different value in each of these calls (within the CreateCluster method i.e.)?

Reply:
It should be the same value. We're only ever executing one of these three calls, so there's no duplication of work.

Reply:
Makes sense.